modyn/selector/internal/selector_strategies/abstract_selection_strategy.py

```diff
@@ -46,6 +46,20 @@ def __init__(
         self.training_set_size_limit: int = config["limit"]
         self.has_limit = self.training_set_size_limit > 0
         self.reset_after_trigger: bool = config["reset_after_trigger"]
+        if "tail_triggers" in config:
+            self.tail_triggers = config["tail_triggers"]
+            assert not (
+                self.reset_after_trigger and self.tail_triggers > 0
+            ), "Reset after trigger is equivalent to setting tail triggers to 0."
+            assert not (
+                not self.reset_after_trigger and self.tail_triggers == 0
+            ), "Reset after trigger is equivalent to setting tail triggers to 0."
```

Comment:
Please use ValueErrors for invalid user input. assertions are for invalid internal state